### PR TITLE
chore: enable renovate automerge for patch/minor versions or devDependencies

### PR DESCRIPTION
**Pull request by raymondfeng**
This PR configures Renovate Bot to automatically merge PRs if:

1. CI is green
2. It updates to a new patch/minor version or only for devDependencies

See https://docs.renovatebot.com/configuration-options/#automerge

Signed-off-by: Raymond Feng <enjoyjava@gmail.com>

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Review by jannyHou — COMMENTED** [review]
> 
    1. CI is green
    2. It updates to a new patch/minor version or only for devDependencies

👍 LGTM
A question: which config in this PR makes sure the CI is green?

**Comment by raymondfeng** [comment]
> A question: which config in this PR makes sure the CI is green?

That's the default behavior of Renovate. It won't auto-merge a PR if CI is not green.

File: .renovaterc.json [hunk 1]
```diff
@@ -5,57 +5,60 @@
...
+    {
+      "packageNames": [
+        "@types/express",
+        "@types/express-serve-static-core"
+      ],
+      "groupName": "@types/express* packages"
+    },
+    {
+      "updateTypes": ["minor", "patch"],
+      "automerge": true
```
Comment (dhmlau) [inline .renovaterc.json:48]: didn't see where we're defining CI needs to be green in order to automerge. Did I miss something?

Comment (raymondfeng) [reply .renovaterc.json:48]: That's built-in with Renovate, which won't merge any PR if CI not green.

**Review by dhmlau — APPROVED** [review]

**Review by jannyHou — APPROVED** [review]

**Comment by bajtos** [comment]
> raymondfeng added the Build  label 2 days ago

So far, we have been using `Internal Tooling` label for all things related to our project infrastructure. Let's keep using that one instead of creating a new `Build` label please.

**Pull request merged**